docusaurus/README.md

```diff
@@ -1,41 +1,45 @@
-# Website
+# Docusaurus
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This [Docusaurus 2](https://docusaurus.io/) project contains the document website of ORT which is published to the
+`gh-pages` branch.
 
-### Installation
+## Installation
 
-```
-$ yarn
+```shell
+npm install
```

Comment:
Just a question: Is the fact that the Docusaurus template uses Yarn a hint that we should have been using Yarn instead of NPM yourselves (for whatever reason)?

Reply:
It's just that the template files are made for Yarn, probably because the project itself also uses Yarn. But the documentation always provides commands NPM, Yarn, and PNPM in that order. I just went with NPM because it's the default for Node, but we can easily switch to Yarn or PNPM later on if there is a reason for that. I think as long as we mainly use the docs plugin it should not make much of a difference which package manager we use.